cmake/CrowConfig.cmake.in

```diff
@@ -28,6 +28,13 @@ check_required_components("@PROJECT_NAME@")
 get_target_property(_CROW_ILL Crow::Crow INTERFACE_LINK_LIBRARIES)
 get_target_property(_CROW_ICD Crow::Crow INTERFACE_COMPILE_DEFINITIONS)
 
+if(_CROW_ILL STREQUAL "_CROW_ILL-NOTFOUND")
```

Comment:
We would not need to set the variables if we later on compare against the ..-NOTFOUND value.